src/main/scala/actors/ForceUpdateActor.scala

```diff
@@ -28,7 +28,7 @@ class ForceUpdateActor(config:Config) extends Actor with CapiCommunicator {
   protected val updater:ActorRef = context.actorOf(Props(new PlutoUpdaterActor(config)))
   protected val unattachedAtomActor:ActorRef = context.actorOf(Props(new UnattachedAtomActor(config)))
 
-  final val client: GuardianContentClient = new GuardianContentClient(config.getString("capi_api_key"))
+  protected def client: GuardianContentClient = new GuardianContentClient(config.getString("capi_api_key"))
```

Comment:
I'd make this a `val`, otherwise a new HTTP client will be created each time it is referenced, with its own thread pool etc. 😨

Reply:
thanks for the headsup, will do. i had made it a val but saw it was originally a def and that was screwing up the test overloads